### ISSUE TITLE
Get-vRACatalogPrincipal wrong with a list of group

### ISSUE THREAD

**Issue by Pisamad** [jakkulabs/PowervRA#272]
Steps to reproduce
------------------
**PS C:\Users\developper> Get-vRACatalogPrincipal -Id 'VRA_grp01@domain.local','VRA_grp02@domain.local' -Verbose**
VERBOSE: Searching for USER VRA_grp01@domain.local
VERBOSE: GET https://vra.domain.local/identity/api/tenants/tenant-rnd/principals/VRA_grp01@domain.local with 0-byte payload
VERBOSE: received 168-byte response of content type application/json
VERBOSE: User VRA_grp01@domain.local not found..
VERBOSE: Searching for GROUP VRA_grp01@domain.local
VERBOSE: GET https://vra.domain.local/identity/api/tenants/tenant-rnd/groups/VRA_grp01@domain.local with 0-byte payload
VERBOSE: received -byte response of content type application/json
VERBOSE: Content encoding: utf-8
VERBOSE: Group found!

VERBOSE: Searching for USER VRA_grp02@domain.local
VERBOSE: GET https://vra.domain.local/identity/api/tenants/tenant-rnd/principals/VRA_grp02@domain.local with 0-byte payload
VERBOSE: received 168-byte response of content type application/json
VERBOSE: User VRA_grp02@domain.local not found..
tenantName ref                               type  value
---------- ---                               ----  -----
tenant-rnd    VRA_grp01@domain.local GROUP
tenant-rnd    VRA_grp01@domain.local GROUP

Expected behavior
-----------------
tenantName ref                               type  value
---------- ---                               ----  -----
tenant-rnd    VRA_grp01@domain.local GROUP
tenant-rnd    VRA_grp02@domain.local GROUP

Actual behavior
---------------
When presenting a list og group to Get-vRACatalogPrincipal, only first group is duplicated

Proposed Solution
-----------------
In function Get-vRACatalogPrincipal, set $CatalogPrincipal = $null in the beginning of the loop :
```powershell
function Get-vRACatalogPrincipal {
[CmdletBinding(DefaultParameterSetName="Standard")][OutputType('System.Management.Automation.PSObject')]
    Param (
[Parameter(Mandatory=$true,ValueFromPipeline=$true,ValueFromPipelineByPropertyName=$true,ParameterSetName="Standard")]
        [ValidateNotNullOrEmpty()]
        [Alias("Principal")]
        [String[]]$Id
    )
    Begin {
    }
    Process {
        try {
            foreach ($PrincipalId in $Id){
                $CatalogPrincipal = $null

                # -- Test for user first
                try {
                    Write-Verbose -Message "Searching for USER $($PrincipalId)"  

```

Environment data
----------------
Powershell 5 and 7

**Comment by Pisamad** [comment]
Sorry, only for power-VRA V3.7.0 !!

**Comment by jonathanmedd** [comment]
Resolved via #273
